src-vue/src/i18n/lang/en.json

```
@@ -146,6 +146,8 @@
     },
 
     "notification": {
+        "date_prefix": "at",
+
         "no_new": {
```

Is the extra new line intentional?
```suggestion
    "notification": {
        "date_prefix": "at",
        "no_new": {
```